From adb0bbc0c3841bc87dbb0d7e7667533586f14320 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 01 Sep 2022 20:19:53 +0300
Subject: [PATCH] [SW-0] preboot: i2ci: please the compiler

The enum hl_fw_fit_boot_error{} is going to be extended with at least
the *_LAST value.
Extending it w/o updating this switch case - produces a compilation
error.
Add default statement to please the compiler.

Change-Id: Ie3e328ebafe1c3edf46ededea32b20f8d61d01c2
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/lib/preboot/i2cslave_indirect.c b/lib/preboot/i2cslave_indirect.c
index f5ee739..a6ee266 100644
--- a/lib/preboot/i2cslave_indirect.c
+++ b/lib/preboot/i2cslave_indirect.c
@@ -1155,6 +1155,8 @@
 		/* set Security error bit of ERR_STS cmd */
 		err_sts_reg_val |= HL_ERROR_STATUS_SECURITY;
 		break;
+	default:	/* please the compiler */
+		break;
 	}
 
 	if (is_update_sig_os) {
